src/vmm/Cargo.toml

```diff
@@ -5,8 +5,14 @@ authors = ["Amazon Firecracker team <firecracker-devel@amazon.com>"]
 edition = "2021"
 license = "Apache-2.0"
 
+[features]
+default = []
+aws-lc-rs = ["dep:aws-lc-rs"]
+rand = ["dep:rand"]
```

Comment:
Either `aws-lc-rs` or `rand` is required. Currently if someone doesn't configure this properly they will get a non-obvious error.

You can add a `build.rs` like:
```rust
fn main() {
    #[cfg(any(
        all(feature="aws-lc-rs", feature="rand"), // If both are enabled
        not(any(feature="aws-lc-rs", feature="rand")) // If neither are enabled
    )]
    compile_error!("Please enable the feature \"aws-lc-rs\" or the feature \"rand\"");
}
```